gradle.properties

```
@@ -1,5 +1,5 @@
 group=io.ballerina.stdlib
-version=1.0.1-SNAPSHOT
+version=1.0.2-SNAPSHOT
```

```suggestion
version=1.1.0-SNAPSHOT
```
Change ballerina/sql as well.